tests/component_handlers/helm_wrapper/test_helm_wrapper.py

```diff
@@ -347,3 +347,169 @@ def test_get_manifest(self, run_command: MagicMock):
 
         run_command.side_effect = ReleaseNotFoundException()
         assert helm_wrapper.get_manifest("test-release", "test-namespace") == ()
+
+    def test_should_call_run_command_method_when_helm_template_with_api_versions(
+        self, run_command: MagicMock
+    ):
+        helm_wrapper = Helm(helm_config=HelmConfig())
+
+        helm_wrapper.template(
+            release_name="test-release",
+            chart="bakdata-streams-bootstrap/streams-app",
+            api_versions="2.1.1",
+        )
+        run_command.assert_called_once_with(
+            [
+                "helm",
+                "template",
+                "test-release",
+                "bakdata-streams-bootstrap/streams-app",
+                "--api-versions" "2.1.1",
+            ],
+        )
+
+    def test_should_call_run_command_method_when_helm_template_without_api_versions(
+        self, run_command: MagicMock
+    ):
+        helm_wrapper = Helm(helm_config=HelmConfig())
+
+        helm_wrapper.template(
+            release_name="test-release",
+            chart="bakdata-streams-bootstrap/streams-app",
+        )
+        run_command.assert_called_once_with(
+            [
+                "helm",
+                "template",
+                "test-release",
+                "bakdata-streams-bootstrap/streams-app",
+            ],
+        )
+
+    def test_helm_template_without_api_versions(self, run_command: MagicMock):
+        helm_wrapper = Helm(helm_config=HelmConfig())
+
+        helm_wrapper.template(
+            release_name="test-release",
+            chart="bakdata-streams-bootstrap/streams-app",
+        )
```

Comment:
What is this test asserting?

Reply:
That the 'implemented' `template` returns what `helm template` would normally return. It is a bit redundant considering that there also is a test that looks if `helm template` is called with the correct values. When I finalize the PR, I would probably either merge this into another test or get rid of it.